components/Starknet/modules/architecture_and_concepts/pages/Network_Architecture/fee-mechanism.adoc

```diff
@@ -81,21 +81,21 @@ The weights are:
 [%autowidth,cols="2,2",options="header",stripes=even]
 |===
 | Step | Gas cost
-| Cairo step | 0.01 gas/step
-| Pedersen | 0.32 gas/application
-| Poseidon | 0.32 gas/application
-| Range check | 0.16 gas/application
-| ECDSA | 20.48 gas/application
-| Keccak | 20.48 gas/application
-| Bitwise | 0.64 gas/application
-| EC_OP | 10.24 gas/application
+| Cairo step | 0.01gwei gas/step
+| Pedersen | 0.32gwei gas/application
+| Poseidon | 0.32gwei gas/application
+| Range check | 0.16gwei gas/application
+| ECDSA | 20.48gwei gas/application
+| Keccak | 20.48gwei gas/application
+| Bitwise | 0.64gwei gas/application
+| EC_OP | 10.24gwei gas/application
 |===
 
 
 [id="_on_chain_data"]
 === Onchain data
 
-The onchain data associated with a transaction is composed of three parts
+The on-chain data associated with a transaction is composed of three parts
```

Comment:
@faytey We had a discussion about _on-chain/off-chain_ vs _onchain/offchain_ and decided to go with _onchain/offchain_. The reasoning was that this is the usage that is trending more and more in the industry.

I just created a style guide PR for this.

If you feel strongly about this usage, I encourage you to open an issue. I'm starting to look at more ways to get the community involved in the docs.

Reply:
Ok since there's been a discussion, i belief it's fine, thanks